samples/meetings-app-icon-badging/nodejs/bots/teamsBot.js

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { TeamsActivityHandler, TeamsInfo, TurnContext, CardFactory } = require("botbuilder");
+var ACData = require("adaptivecards-templating");
+const notificationCardJson = require('../resources/SendTargetNotificationCard.json');
+
+class TeamsBot extends TeamsActivityHandler {
+    constructor() {
+        super();
+        this.baseUrl = process.env.BaseUrl;
+        this.AppId = process.env.MicrosoftAppId;
+
+        // This method is invoked whenever there is any message activity in bot's chat.
+        this.onMessage(async (context, next) => {
+            var members = new Array();
+            let meetingId = context.activity.channelData.meeting.id;
+
+            if (context.activity.value == null) {
+                TurnContext.removeRecipientMention(context.activity);
+
+                if (context.activity.text.trim() == "SendNotification") {
+                    var meetingMembers = await TeamsInfo.getPagedMembers(context);
+                    let tenantId = context.activity.channelData.tenant.id;
+
+                    for (var member in meetingMembers) {
+                        let participantDetail = await TeamsInfo.getMeetingParticipant(context, meetingId, member.aadObjectId, tenantId);
+
+                        // Select only those members that present when meeting is started.
+                        if (participantDetail.meeting.inMeeting) {
+                            members.push({ id: participantDetail.user.id, name: participantDetail.user.name })
+                        }
+                    }
+
+                    // Send and adaptive card to user to select members for sending targeted notifications.
```

Comment:
nit typo - Send an adaptive card to the user to select members for sending targeted notifications.